lib/rules/no-unused-refs.js

```diff
@@ -233,6 +233,23 @@ module.exports = {
               refsNode = id.parent
             }
             extractUsedForPattern(refsNode)
+          },
+          CallExpression(exp) {
+            if (exp.callee.name !== 'useTemplateRef') {
+              return
+            }
+            const arg0 = exp.arguments[0]
+            if (!arg0) {
+              return
+            }
+            if (arg0.type === 'Literal') {
+              usedRefs.add(arg0.value)
```

Comment:
Could you please use `utils.getStringLiteralValue()`? Please refer to the code of custom-event-name-casing for usage.

https://github.com/vuejs/eslint-plugin-vue/blob/d77fbf7a05f3a95085ddbfeec3eaee576e35ecd7/lib/rules/custom-event-name-casing.js#L32